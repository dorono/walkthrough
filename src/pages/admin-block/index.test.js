import React from 'react';
import {shallow} from 'enzyme';
import {AdminBlockPage} from './index';

const defaultProps = {
    data: {},
    location: {},
};

describe('AdminBlock', () => {
    it('should render without errors', () => {
        shallow(<AdminBlockPage {...defaultProps} />);
    });

    it('should match snapshot', () => {
        expect(shallow(<AdminBlockPage {...defaultProps} />)).toMatchSnapshot();
    });
});
