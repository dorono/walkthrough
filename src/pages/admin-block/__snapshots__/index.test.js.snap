// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`AdminBlock should match snapshot 1`] = `
<div>
  <Container
    primary={true}
    showFullWidthBanner={false}
    title="Admin block"
  >
    <Vertical>
      <Box
        type="outline"
      >
        <Label>
          HASH
        </Label>
        <withRouter(Hash)
          type="ablock"
        />
      </Box>
      <Box
        type="fill"
      >
        <Label>
          PARENT DIRECTORY BLOCK
        </Label>
        <DirectoryBlockLink />
      </Box>
    </Vertical>
  </Container>
  <DataLoader
    contentColumnName="CONTENT"
    entriesUrl="/ablocks/undefined/entries"
    hasLink={false}
    renderContent={[Function]}
  />
</div>
`;
